src/components/navigation/Navigation.tsx

```diff
@@ -66,19 +66,21 @@ const Navigation = (props: PropTypes.InferProps<typeof NavigationProps>) => {
                         link="/app/captures"
                         key="Capture"
                     />
-                    <ListItemLink
-                        icon={<CodeIcon />}
-                        title="Derivations"
-                        link="/app/derivations"
-                        key="Derivations"
-                        isOpen={open}
-                    />
                     <ListItemLink
                         icon={<StorageIcon />}
                         title="Materializations"
                         link="/app/materializations"
                         key="Materializations"
                         isOpen={open}
+                        disabled={true}
+                    />
+                    <ListItemLink
+                        icon={<CodeIcon />}
+                        title="Derivations"
+                        link="/app/derivations"
+                        key="Derivations"
+                        isOpen={open}
+                        disabled={true}
```

Comment:
Not important (for now) but I feel like we should order the navigation in order of how folks might use them? Not sure. So I feel like you'd setup a capture and materialization first.